src/client/theme-default/components/VPHome.vue

```diff
@@ -12,6 +19,14 @@ import VPHomeFeatures from './VPHomeFeatures.vue'
     <slot name="home-features-before" />
     <VPHomeFeatures />
     <slot name="home-features-after" />
+
+    <div class="content">
+      <div class="content-container">
+        <main class="main">
+          <Content class="vp-doc" :class="pageName" />
```

Comment:
We probably don't need this `pageName` feature here. People usually don't have multiple home pages. This was added in docs so that people may customize css for specific page.